### DISCUSSION TITLE
Feature request: display current window/region resolution

### DISCUSSION THREAD

**Discussion by kuoirad** [tom-englert/RegionToShare#10] [Ideas]
While I know you use Fancy Zones, not all of us do.  It would be helpful to have the current resolution of the region/window to be displayed somewhere so we can tailor that size more accurately.

Thanks!  I just joined the "ultrawide religon" and this appears to be the best solution I've found to work around Teams's inability to share part of a screen.

**Comment by tom-englert** [comment]
Added in latest version
